code/game/objects/items/implants/implant_mindshield.dm

```diff
@@ -57,7 +57,7 @@
 			remove_hivemember(target)
 
 		if(woke)
-			woke.one_mind.remove_member(target.mind)
+			woke.one_mind?.remove_member(target.mind)
```

Comment:
?

Reply:
the `one_mind` variable may be null even if they are an antag and i forgot the safety
and yes, `?.` is somehow an actual operator

Reply:
I don't like it

Reply:
>use `if` on a PR
>cyberboss says use `?.`
>use `?.` on next PR
>oranges says use `if`

the duality of man